pyramid_swagger/exceptions.py

```diff
@@ -2,10 +2,10 @@
 import sys
 
 import six
-from pyramid.httpexceptions import HTTPClientError, HTTPInternalServerError
+from pyramid.httpexceptions import HTTPBadRequest, HTTPInternalServerError
 
 
-class RequestValidationError(HTTPClientError):
+class RequestValidationError(HTTPBadRequest):
```

Comment:
Great. I prefer to just use the right exception class, although this is technically an interface change I'm really really hoping any users who care are directly catching RequestValidationError, not the inherited error class. 

Feel like something where a bugfix bump to the version with a clear CHANGELOG is sufficient warning?